login.css

```diff
@@ -33,7 +33,7 @@ ul{
 }
 /* Set up position for the text in nav to the right */
 li{
-	float:left;
+	float:right;
```

Comment:
12/08/2021: Make the link in the nav bar to be in the right side. Because I released if the nav bar was all in the left side then it is look a bit empty so I will make the logo to be in the left and the link to be in the right side so the navigation bar to look more balance.